backends/edk2-compat/edk2-svc-validate.c

```diff
@@ -119,8 +119,8 @@ static int parse_opt(int key, char *arg, struct argp_state *state)
 		verbose = PR_DEBUG;
 		break;
 	// set varname as dbx, important for validating ESL
-	case 'x':
-		args->varName = "dbx";
+	case 'n':
+		args->varName = arg;
```

Comment:
If I'm reading this correctly, this also changed the generate command for dbx now, right?
So sbat is:
```
./secvarctl generate f:a -k <signerPrivate.key> -c <signerPublic.crt> -i <sbatFile> -o <out.auth> -n sbat
```
and now dbx will be
```
./secvarctl generate f:a <args> -n dbx
```
correct?

This is a small change in the argument usage that would be breaking for any scripts, but I do like the more unified approach over having a specialized argument for only `dbx`.

@nick-child-ibm any thoughts on this?